apps/studio/src/lib/db/clients/utils.ts

```diff
@@ -98,12 +98,14 @@ export function buildFilterString(filters: TableFilter[], columns = []) {
           : '?'
 
         return `${field} ${item.type.toUpperCase()} (${questionMarks})`
+      } else if (item.type.includes('is')) {
```

Comment:
This needs integration tests for the drivers to make sure they process and query themselves correctly